flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorBaseTest.java

```diff
@@ -97,29 +94,33 @@
 import static org.hamcrest.Matchers.samePropertyValuesAs;
 import static org.junit.Assert.assertTrue;
 
-/** IT cases for all cassandra sinks. */
+/**
+ * Base class for IT cases for all Cassandra sinks. This class relies on Cassandra testContainer
+ * that needs to use a ClassRule. Parametrized tests to not work with ClassRules so the actual
+ * testCase classes define the tested version and manage the container.
+ */
 @SuppressWarnings("serial")
 // NoHostAvailableException is raised by Cassandra client under load while connecting to the cluster
 @RetryOnException(times = 3, exception = NoHostAvailableException.class)
-public class CassandraConnectorITCase
+public abstract class CassandraConnectorBaseTest
```

Comment:
This isn't a test, and hence the name shouldn't end in "Test". switch "Base" and "Test".

Reply:
:+1: